packages/twenty-front/src/testing/decorators/getFieldDecorator.tsx

```diff
@@ -34,6 +39,14 @@ const RecordMockSetterEffect = ({
     [setRecordValue],
   );
 
+  useEffect(() => {
```

Comment:
This feels off to me, are you sure it's needed? Why are you setting the date time format here? Can't you set it once only in when after login/workspaceMember update?

Reply:
I'm sorry my comment was stupid, didn't notice it was a test. Just saw "getFieldDecorator" and then that useEffect within it and jumped to a wrong conclusion

Reply:
haha no worries! XD